src/main/scala/uk/gov/hmrc/hmrcfrontend/views/html/helpers/HmrcLayout.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcfrontend.views.html.helpers
+
+import play.api.i18n.Messages
+import play.api.mvc.RequestHeader
+import play.twirl.api.{Html, HtmlFormat}
+import uk.gov.hmrc.govukfrontend.views.Aliases.{BackLink, PageLayout, PhaseBanner}
+import uk.gov.hmrc.govukfrontend.views.html.components.{FixedWidthPageLayout, TwoThirdsMainContent}
+import uk.gov.hmrc.hmrcfrontend.views.Aliases.Header
+
+import javax.inject.Inject
+
+class HmrcLayout @Inject() (
+  deprecatedHmrcLayout: DeprecatedHmrcLayout,
+  twoThirdsMainContent: TwoThirdsMainContent,
+  fixedWidthPageLayout: FixedWidthPageLayout
+) {
+  @deprecated(
+    "Use HmrcStandardPage template instead - see DeprecatedHmrcLayout for details of how HmrcLayout parameters map to HmrcStandardPage",
+    "6.3.0"
```

Comment:
is it OK to leave this as the original version this was deprecated in?  might surprise teams on a later version who upgrade to this and start getting warnings

Reply:
I think it's ok to leave it - it's deprecated but there's schedule for when we're going to remove it